src/main/java/uk/gov/hmcts/reform/juddata/camel/util/MappingConstants.java

```diff
@@ -67,4 +67,7 @@ private MappingConstants() {
 
     public static final String ERROR_MESSAGE = "ErrorMessage";
 
+    public static final String INVALID_JSR_PARENT = "Skipped record as it is invalid in parent";
+
```

Comment:
can u change message to - 
"Record skipped due to jsr violation in the record in the parent load"

Reply:
done